### PR TITLE
Create component browser in new UI sample app, add AvatarView screen

### PR DESCRIPTION
**Pull request by zsmb13**
### Description

Adds a Component Browser screen where we can try our new UI components in different configurations. Accessible via a long tap of the version number.

<p float="left">
    
    
</p>

### Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- [ ] ~Changelog updated with client-facing changes~
- [ ] ~New code is covered by unit tests~
- [x] Comparison screenshots added for visual changes
- [x] Reviewers added

### CODE REVIEW

File: stream-chat-android-ui-components-sample/src/main/res/layout/fragment_component_browser_avatar_view.xml
Comment (adasiewiczr) [inline stream-chat-android-ui-components-sample/src/main/res/layout/fragment_component_browser_avatar_view.xml:12]: Our `dimens.xml` contains some commonly used spacings (e.g. 8, 16, 24, etc.). Maybe let's reuse them?

**Review by adasiewiczr — APPROVED** [review]

File: stream-chat-android-ui-components-sample/src/main/res/layout/fragment_component_browser_avatar_view.xml
Comment (zsmb13) [reply stream-chat-android-ui-components-sample/src/main/res/layout/fragment_component_browser_avatar_view.xml:12]: Do we expect to change those spacings at some point, and when that happens, for all of their usages to change together?

I'm always sceptical about dimension constants, because they often just mean "24dp" and not "this same medium padding everywhere it's used". They're rarely a good abstraction.

Comment (adasiewiczr) [reply stream-chat-android-ui-components-sample/src/main/res/layout/fragment_component_browser_avatar_view.xml:12]: 👍 good point, I don't think we can expect such changes

File: stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/component_browser/home/ComponentBrowserHomeFragment.kt
Comment (JcMinarro) [inline stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/component_browser/home/ComponentBrowserHomeFragment.kt:24]: It shouldn't be needed after we merge #141

**Review by JcMinarro — CHANGES_REQUESTED** [review]
Looks great.
Wait until #829 is merged and update it removing the `ChatInitializer` initialization

**Review by JcMinarro — APPROVED** [review]

**Pull request merged**
